build.gradle

```diff
@@ -28,10 +28,9 @@ ext {
     ballerinaJreArtifactsVersion = "2.0.0"
     dockerVersion = project.ballerinaVersion
     kubernetesVersion = project.ballerinaVersion
-    dataMapperVersion = project.ballerinaVersion
     awslambdaVersion = project.ballerinaVersion
     picocliVersion = "4.0.1"
-    ballerinaCommandVersion = "0.8.6-SNAPSHOT"
+    ballerinaCommandVersion = "0.8.7"
```

Comment:
Shouldn't this have been bumped to the next development version?

Reply:
This is not released by pipeline. It was added as a dependency separately therefore we need to update manually